ts/features/bonus/cdc/components/CdcServiceCTA.tsx

```diff
@@ -104,19 +113,52 @@ const ErrorButton = () => {
 const CdcServiceCTAButton = () => {
   const dispatch = useIODispatch();
   const cdcBonusRequestList = useIOSelector(cdcBonusRequestListSelector);
+  const allAvailableBonusTypes = useIOSelector(allAvailableBonusTypesSelector);
+  const cdcInfo = useIOSelector(availableBonusTypesSelectorFromId(ID_CDC_TYPE));
+
+  useFocusEffect(
+    useCallback(() => {
+      if (cdcInfo === undefined) {
+        dispatch(loadAvailableBonuses.request());
+      }
+    }, [cdcInfo, dispatch])
+  );
 
   useFocusEffect(
     useCallback(() => {
       dispatch(cdcRequestBonusList.request());
     }, [dispatch])
   );
 
-  return fold(
-    cdcBonusRequestList,
+  return pot.fold(
+    allAvailableBonusTypes,
     () => null,
     () => <ActivityIndicator />,
-    bonusRequestList => <ReadyButton bonusRequestList={bonusRequestList} />,
-    _ => <ErrorButton />
+    () => <ActivityIndicator />,
+    () => (
+      <ErrorButton onPress={() => dispatch(loadAvailableBonuses.request())} />
+    ),
+    _ => {
+      if (cdcInfo === undefined) {
```

Comment:
```suggestion
      if (
        cdcInfo === undefined ||
        cdcInfo?.visibility === BonusVisibilityEnum.hidden
      ) {
```
If I understand the required behaviour, the cta should be rendered only if:
- local ff is true
- remote ff is true
- The bonus exists in the bonus_list and is visible

While the first two conditions are ensured by the `SpecialServicesCTA` component, the last one must be made explicit, otherwise when the bonus visibility is `hidden`, the cta would be rendered, without following the same semantic of the bonus list.
What you think?

Reply:
Sure, thank you! Updated in [375d6ce](https://github.com/pagopa/io-app/pull/3970/commits/375d6ce3ca8228bbd51a56ecfe973f36ee691879)